backend/app/node_handler.go

```diff
@@ -491,3 +498,67 @@ func (h *Handler) getRentedNodesForUser(ctx context.Context, userID int, healthy
 
 	return nodes, count, nil
 }
+
+// @Summary Get account ID by twin ID
+// @Description Retrieve the account ID associated with a specific twin ID
+// @Tags twins
+// @Accept json
+// @Produce json
+// @Param twin_id path int true "Twin ID"
+// @Param limit query int false "Pagination limit"
+// @Param offset query int false "Pagination offset"
+// @Param filterParam  query string false "Other optional filter params"
+// @Success 200 {object} AccountIDResponse "Account ID is retrieved successfully"
+// @Failure 400 {object} APIResponse "Bad Request or Invalid params"
+// @Failure 404 {object} APIResponse "Twin ID not found"
+// @Failure 500 {object} APIResponse "Internal Server Error"
+// @Router /twins/{twin_id}/account [get]
+func (h *Handler) GetAccountIDHandler(c *gin.Context) {
+	twinIDParam := c.Param("twin_id")
+	if twinIDParam == "" {
+		Error(c, http.StatusBadRequest, "Twin ID is required", "")
+		return
+	}
+
+	query := c.Request.URL.Query()
+
+	limit := proxyTypes.DefaultLimit()
```

Comment:
We only need the twin filter, no need for limits nor pagination correct? it's only one twin afterall